### PR TITLE
Make Rectangle's point fields public.

### PR DESCRIPTION
**Pull request by PaulDGillis**
I'm working on an app that needs to create a window above or below the tray icon. Couldn't find another way to consistently place the window without making these public.

### CODE REVIEW

**Review by amrbashir — CHANGES_REQUESTED** [review]
Thanks for making this PR, could you add a change file in `.changes` folder? see `.changes/readme.md` for guidance

**Comment by PaulDGillis** [comment]
> Thanks for making this PR, could you add a change file in `.changes` folder? see `.changes/readme.md` for guidance

Should be fixed, didn't see a contributing readme, thanks!

File: .changes/public.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"tray-icon": "minor"
```
Comment (amrbashir) [inline .changes/public.md]:
```suggestion
"tray-icon": "patch"
```

**Review by amrbashir — APPROVED** [review]
Thanks

**Pull request merged**